libs/workorderapp/package.json

```diff
@@ -12,6 +12,7 @@
     "@cc-components/shared": "workspace:^",
     "@cc-components/sharedcomponents": "workspace:^",
     "@cc-components/workordershared": "workspace:^",
-    "@cc-components/workordersidesheet": "workspace:^"
+    "@cc-components/workordersidesheet": "workspace:^",
+    "@cc-components/modelviewer": "workspace:^"
```

Comment:
Why would the Workorder app be dependent on modelviewer? Isnt 3d only in the sidesheet?

Reply:
Agree. Removed